client/src/pages/Dashboard.js

```diff
@@ -0,0 +1,291 @@
+  return (
+    <Box mb={8}>
+      <Container>
+        <AppBar variant="outlined" className={classes.appBar} position="fixed">
+          <Toolbar>
+            <img
+              className={classes.logo}
+              src={logo}
+              alt="Product Launch Logo"
+            ></img>
+            <Typography variant="h6" className={classes.title} component="h1">
+              Product Launch
+            </Typography>
+            <Button color="inherit">Explore</Button>
+            <Button color="inherit">Launch</Button>
+            <Button color="inherit">Login</Button>
+          </Toolbar>
+        </AppBar>
+      </Container>
+    </Box>
+  );
+}
+
+function UserInfo(props) {
+  const classes = useStyles();
+  return (
+    <Box height="100%" className={classes.userInfoShadow}>
+      <Container align="center">
+        {/* Avatar */}
+        <Avatar className={classes.avatar} src={profpic1}></Avatar>
+        {/* User Info */}
+        <Box>
+          <Typography variant="h6" component="p">
+            Alexander Faa
+          </Typography>
+          <Typography color="textSecondary">New York, NY</Typography>
+        </Box>
+
+        {/* Send a message */}
+        <Box className={classes.ySpacing}>
+          <Button
+            className={classes.sendMessage}
+            size="large"
+            variant="outlined"
+            disableElevation
+          >
+            Send a Message
+          </Button>
+        </Box>
+
+        {/* Description */}
+        <Box className={classes.ySpacing}>
+          <Typography>
+            I just have a great passion for all things coffee
+          </Typography>
+        </Box>
+
+        {/* Expertise */}
+        <Box marginTop={2}>
+          <Typography fontWeight="fontWeightMedium">Expertise</Typography>
+          <Button
+            className={classes.roundButton}
+            variant="outlined"
+            size="small"
+          >
+            Marketing
+          </Button>
+          <Button
+            className={classes.roundButton}
+            variant="outlined"
+            size="small"
+          >
+            Coffee
+          </Button>
+          <Button
+            className={classes.roundButton}
+            variant="outlined"
+            size="small"
+          >
+            Technology
+          </Button>
+        </Box>
+      </Container>
+
+      <Divider className={classes.ySpacing} />
+
+      {/* Looking to Invest in */}
+      <Box>
+        <Container align="center">
+          <Typography fontWeight="fontWeightMedium">
+            Looking to invest in
+          </Typography>
+
+          <Button
+            className={classes.highlightButton}
+            variant="outlined"
+            size="small"
+          >
+            Coffee
+          </Button>
+        </Container>
+      </Box>
+    </Box>
+  );
+}
+
+function ProjectCard(props) {
+  const classes = useStyles();
+  console.log(props.ver);
+  const projectStatic = [
```

Comment:
you might want to move this static projects list outside of this container, and map through each object and pass it on as a prop

Reply:
That's a good idea. I'll make that change